config/pipeline.yaml

```
@@ -117,6 +117,18 @@ _anchors:
       - 'amlogic'
       - 'ardb'
       - 'arnd'
+      - 'clk'
+      - 'peterz'
+
+  build-only-trees: &build-only-trees-rules-minimal
```

```suggestion
  build-only-trees-minimal: &build-only-trees-minimal-rules
```

The entry has the same name as the previous one (`build-only-trees`) which should be avoided. Moreover, then anchor name should end with the "kind", in this case `-rules`.

I'm not a fan of having that many duplicated entries. If it's really necessary to have 2 sets of build-only trees, I'd rather have those as separate entries; although it feels better to have a single set and simply exclude `efi` adding a simple rule to scheduler entries where it shouldn't be used.

---

I'd not thought of it like this. I've updated as you have asked.